resources/views/docs/_structures/score.md

```diff
@@ -2,37 +2,39 @@
+is_perfect_combo    | boolean                             | Whether the user has achieved perfect combo using lazer logic.<sup>1</sup>
+legacy_perfect      | boolean                             | Whether the user has achieved perfect combo using stable logic.<sup>1</sup>
+legacy_score_id     | integer?                            | Only present on scores set in stable. The ID of this score in the old ID scheme preceding lazer.
+legacy_total_score  | integer                             | Only present on scores set in stable. The total score achieved by the user in stable's scoring algorithm (colloquially known as "score V1").
+max_combo           | integer                             | The largest combo achieved by the user during gameplay.
+maximum_statistics  | [ScoreStatistics](#scorestatistics) | The best possible score statistics achievable on the map the user played.
+mods                | [Mod](#mod)[]                       | The mods used when setting this score.
+passed              | boolean                             | Whether the user passed or failed gameplay.
+playlist_item_id    | integer                             | The ID of the multiplayer playlist item associated with this score. Only present for multiplayer scores.
+pp                  | float?                              | Number of performance points awarded to this score.
+preserve            | boolean                             | Whether or not the score may eventually be deleted. Only present for `solo_score` type.
+processed           | boolean                             | Whether or not the score has been fully processed server-side (most notably including calculation of `pp`). Only present for `solo_score` type.
+rank                | string                              | The letter rank achieved by the score. Notably, this always follows [lazer's updated rank criteria](https://osu.ppy.sh/wiki/en/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)#differences-in-grading-systems).
+ranked              | boolean                             | Whether or not this score can be visible on beatmap leaderboards. Notably, presence of unranked mods **does not** influence the value of this flag, and whether a score gives PP **does not** depend on this flag. Only for `solo_score` type.
+room_id             | integer                             | The ID of the room that the user was in when setting this score. Only present for multiplayer scores.
+ruleset_id          | integer                             | The ID of the ruleset that this score was set on.
+started_at          | [Timestamp](#timestamp)?            | When the user started playing.
+statistics          | [ScoreStatistics](#scorestatistics) | The score statistics achieved in this score,
+total_score         | integer                             | Total score calculated using lazer's "standardised scoring" variant.
+type                | string                              |
```

Comment:
that said, the closing `|` is still needed when it's empty to match the column count 👀